mantidproject/mantid
# MaxEnt not outputing evolution of chi and MaxAngle once it has converged

This is to prevent MaxEnt from outputting workspaces containing mostly zeros. Instead we do not return the values after the algorithm has converged. 

**To test:**
run the following script and look at the data for evolChi1 and evolAngle1. Do this with both this version and master (at the same time). The last entry into the workspaces for this PR should be followed by zeros in master. 

```
from datetime import datetime 

Load(Filename=r'EMU00020884.nxs', OutputWorkspace='ws')
CropWorkspace(InputWorkspace='ws', OutputWorkspace='ws', XMin=0.17, XMax=15, EndWorkspaceIndex=10)
ws = RemoveExpDecay(InputWorkspace='ws')
ws = Rebin(InputWorkspace='ws', Params='0.016')
startTime= datetime.now() 
evolChi1, evolAngle1, image1, data1 = MaxEnt(InputWorkspace='ws', A=0.0001, MaxIterations=2500, ResolutionFactor=1, MaxAngle=0.05)
timeElapsed=datetime.now()-startTime 

print('Time elpased (hh:mm:ss.ms) {}'.format(timeElapsed))
```




Fixes #21392. 

**Release Notes** 

In release notes, also added to the release notes for #20983 
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
